docs/en/docs/sql-manual/sql-functions/bitwise-functions/bitshfitleft.md

```diff
@@ -0,0 +1,71 @@
+---
+{
+"title": "BIT_COUNT",
+"language": "en"
```

Comment:
this  title need change to bit_shift_left
and the zh-CN docs also not match

Reply:
> this title need change to bit_shift_left and the zh-CN docs also not match

I will fix this